### PR TITLE
VACMS-13664 Fix broken Facility force push post API code.

### PR DESCRIPTION
**Pull request by tonytaylor**
## Description

Undoes nodeQuery method calls in buildForm.

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As an admin
1. Navigate to https://pr14287-uzdmvfvvqkhzduhi63guxovruvfrwzgg.ci.cms.va.gov/admin/config/va-gov-post-api/facility-force-queue
   - [x] Validate that each items have counts that are not 0
2. Then Select the option "VBA Facilities"
3. Click the "Force Queue Selected items" button
4. When the page returns scroll down the list of queued items
   - [x] Validate that the count of items queues matches the count of items processed 
   `451 vba_facility nodes processed. Queued 451 for sync to Lighthouse.`
image

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php [hunk 1]
```diff
@@ -77,33 +77,39 @@ public function getFormId() {
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
     // Queries to get total number of nodes for each type for reference.
-    $health_care_local_facility = $this->nodeQuery
+    $health_care_local_facility = $this->entityTypeManager
+      ->getStorage('node')->getQuery('AND')
```
Comment (timcosgrove) [inline docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php:81]: I realize this is what I typed in Slack, but properly this should be on two lines: 
```
->getStorage('node')
->getQuery('AND')
```

File: docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php [hunk 2]
```diff
@@ -164,7 +170,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ->set('bypass_data_check', 1)
       ->save();
 
-    $sandbox['nids'] = $this->nodeQuery
+    $sandbox['nids'] = \Drupal::entityQuery('node')
```
Comment (timcosgrove) [inline docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php:173]: This one should be changed as well.

**Review by timcosgrove — CHANGES_REQUESTED** [review]
1. These three also need changes: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14287/files#diff-5c363b3d533d0a40413daf977c9c8b3152b82139faefb3b2eddb819f87ea19f2R116-R129

2. This is no longer necessary and should come out: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14287/files#diff-5c363b3d533d0a40413daf977c9c8b3152b82139faefb3b2eddb819f87ea19f2R30-R35

3. Same with this: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14287/files#diff-5c363b3d533d0a40413daf977c9c8b3152b82139faefb3b2eddb819f87ea19f2R45-R47

And then, like I mentioned inline, the added line should properly be two lines.

**Comment by swirtSJW** [comment]
Something seems off with the commit history.  There are 23 commits on this branch when there should at most be 2.  It seems like something is definitely not set up correctly on your git configuration or practice.

File: docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php (hunk 1 above)
Comment (tonytaylor) [reply docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php]: agreed. done.

File: docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php (hunk 2 above)
Comment (tonytaylor) [reply docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php]: Yep.

**Comment by tonytaylor** [comment]
@timcosgrove @swirtSJW Thanks for this. I've updated as directed and I'll review my configuration a bit later this morning.

**Comment by swirtSJW** [comment]
This change looks good and tests out but we want to get all these extraneous commits out of here 
image


I am happy to help you sort this out @tonytaylor

**Review by swirtSJW — APPROVED** [review]
Tests out great.  Queing of all facility types works as well as Health services.

**Pull request merged**